content/topics/dotenv-module/_index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Introduction to Dotenv module 
+ready: true
+
+---
+
+## What is an environment variable?
+
+An [environment variable](https://en.wikipedia.org/wiki/Environment_variable) is a variable whose value is set outside the program, typically through functionality built into the operating system or microservice. An environment variable is made up of a name/value pair, and any number may be created and available for reference at a point in time.
```

Comment:
Rather don't mention micro-service like this. I think you really mean to say 'container' but the container is effectively running an operating system. 

We also have some content on environmental variables already. I would add something like this:

```
For a more in-depth description of environmental variables, take a look at this: {{% contentlink "topics/linux/os-environmental-variables/" %}}
```